theme/template-parts/blocks/navigation-link.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @package MaterialDesign
+ */
+
+/**
+ * Template part for displaying individual navigation links.
+ */
+
+$block                  = isset( $args['block'] ) ? $args['block'] : [];
+$attributes             = isset( $args['attributes'] ) ? $args['attributes'] : [];
+$content                = isset( $args['content'] ) ? $args['content'] : [];
+$navigation_link_has_id = isset( $attributes['id'] ) && is_numeric( $attributes['id'] );
+$is_post_type           = isset( $attributes['kind'] ) && 'post-type' === $attributes['kind'];
+$is_post_type           = $is_post_type || isset( $attributes['type'] ) && ( 'post' === $attributes['type'] || 'page' === $attributes['type'] );
+
+// Don't render the block's subtree if it is a draft or if the ID does not exist.
+if ( $is_post_type && $navigation_link_has_id ) {
+	$linked_post = get_post( $attributes['id'] );
+	if ( ! $linked_post || 'publish' !== $linked_post->post_status ) {
+		return '';
+	}
+}
+
+// Don't render the block's subtree if it has no label.
+if ( empty( $attributes['label'] ) ) {
+	return '';
+}
+
+$colors     = block_core_navigation_link_build_css_colors( $block->context, $attributes );
+$font_sizes = block_core_navigation_link_build_css_font_sizes( $block->context );
+$classes    = array_merge(
+	$colors['css_classes'],
+	$font_sizes['css_classes']
+);
+
+$style_attribute = ( $colors['inline_styles'] . $font_sizes['inline_styles'] );
+
+$css_classes = trim( implode( ' ', $classes ) );
+$has_submenu = count( $block->inner_blocks ) > 0;
+$is_active   = ! empty( $attributes['id'] ) && ( get_the_ID() === $attributes['id'] );
+
+$wrapper_attributes = get_block_wrapper_attributes(
+	[
+		'class' => $css_classes . ' wp-block-navigation-item' . ( $has_submenu ? ' has-child' : '' ) .
+			( $is_active ? ' current-menu-item' : '' ),
```

Comment:
Should this be `.mdc-tab--active`?  When visiting active tab, it's not highlighting for me.